### PR TITLE
Remove party lib

### PR DESCRIPTION
**Pull request by tomaszwojcikowski**

### CODE REVIEW

**Comment by tomaszwojcikowski** [comment]
Dialyzer is busted, but it is the same in master, so...

File: Makefile [hunk 1]
```diff
@@ -30,3 +30,6 @@ dialyzer:
 
 xref:
 	$(R3) xref
+
+priv/aws_credentials.term:
```
Comment (maxkuzmins) [inline Makefile]: Never seen a make target with slash and dot in it's name. Can it cause problems? :thinking:

Comment (legoscia) [reply Makefile]: Should be fine - it names a file that this rule can create.

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,14 @@
 language: erlang
+dist: xenial
 sudo: false
 notifications:
      email:
          - backend@gameanalytics.com
 otp_release:
-    - 19.3
```
Comment (lastres) [inline .travis.yml]: since this is a public repository, can we build it in Travis against some other OTP versions if it is indeed backwards compatible?

Comment (tomaszwojcikowski) [reply .travis.yml]: I will add it.

**Pull request merged**